corehq/apps/reports/standard/cases/utils.py

```diff
@@ -115,7 +115,15 @@ def get_case_owners(request, domain, mobile_user_and_group_slugs):
                     )).fields(["users"])
             )
             user_lists = [group["users"] for group in report_group_q.run().hits]
-            selected_reporting_group_users = list(set().union(*user_lists))
+            selected_reporting_group_users = set()
```

Comment:
@zandre-eng Just to make sure I understand what is being returned from ES, when you say

> string of the user ID and not a list containing the user ID

do you mean ES returns a single user like
> "<user_id>"

instead of

> ["<user_id>"]

?

Reply:
That is correct @Charl1996. For example, if we have group A with user 1 and group B with user 2 and 3, the query will be returned as follows (assuming both groups are selected for filtering):
```
res = [
    "1",
    ["2", "3"]
]
```